File: Programmers/lv1/lv1_시저 암호.java
https://programmers.co.kr/learn/courses/30/lessons/12926

import java.util.*;

class Solution {
    public String solution(String s, int n) {
        String answer = "";
        StringBuilder sb = new StringBuilder();

		for (int i = 0; i < s.length(); i++) {
			char ch = s.charAt(i);

			if (ch >= 'a' && ch <= 'z') {
				ch = (char) (ch + n);
				if (ch > 'z') ch -= 26;
			} else if (ch >= 'A' && ch <= 'Z') {
				ch = (char) (ch + n);
				if (ch > 'Z') ch -= 26;
			}
            
			sb.append(ch);
		}
        answer = sb.toString();
        
		return answer;
    }
}

File: Programmers/C#/lv1/lv1_시저 암호.cs
https://programmers.co.kr/learn/courses/30/lessons/12926?language=csharp

public class Solution
{
    public string solution(string s, int n)
    {
        string answer = "";

        char[] arr = s.ToCharArray();
        for (int i = 0; i < s.Length; i++)
        {
            int m = arr[i];
            if (m >= 'a' && m <= 'z')
                answer += (char)('a' + (m - 'a' + n) % 26);
            else if (m >= 'A' && m <= 'Z')
                answer += (char)('A' + (m - 'A' + n) % 26);
            else
                answer += ' ';
        }

        return answer;
    }
}